rackcity/utils/rackcity_utils.py

```diff
@@ -73,10 +73,12 @@ def validate_asset_location(
 
     for asset_in_rack in assets.filter(rack=rack_id):
         # Ignore if asset being modified conflicts with its old location
+
         is_valid_conflict = asset_id is None or asset_in_rack.id != asset_id
-        if change_plan:
+        print("location", is_valid_conflict,asset_in_rack.id, asset_id)
+        if related_asset_id:
             is_valid_conflict = (
-                related_asset_id is not None and asset_in_rack.id != related_asset_id
+                asset_in_rack.id != related_asset_id
```

Comment:
if you are on a change plan but there is not related_asset_id, then we are still validating the location of the change on the "live asset"